@xen-orchestra/web-core/lib/components/ui/top-bottom-table/UiTopBottomTable.vue

```diff
@@ -0,0 +1,64 @@
+<!-- v2 -->
+<template>
+  <div class="ui-top-bottom-table">
+    <div class="content">
+      <span class="typo p3-regular label">
+        {{ $t('core.select.n-selected-of', { count: selectedItems, total: totalItems }) }}
+      </span>
+
+      <UiButton
+        v-if="selectedItems < totalItems"
+        accent="info"
+        size="small"
+        variant="tertiary"
+        @click="emit('toggleSelectAll', true)"
+      >
+        {{ $t('core.select.all') }}
+      </UiButton>
+      <UiButton
+        v-if="selectedItems > 0"
+        accent="info"
+        size="small"
+        variant="tertiary"
+        @click="emit('toggleSelectAll', false)"
+      >
+        {{ $t('core.select.unselect') }}
+      </UiButton>
```

Comment:
Following Clémence's suggestion, we will opt to disable buttons rather than removing them.

```suggestion
      <UiButton
        :disabled="selectedItems === totalItems"
        accent="info"
        size="small"
        variant="tertiary"
        @click="emit('toggleSelectAll', true)"
      >
        {{ $t('core.select.all') }}
      </UiButton>
      <UiButton
        :disabled="selectedItems === 0"
        accent="info"
        size="small"
        variant="tertiary"
        @click="emit('toggleSelectAll', false)"
      >
        {{ $t('core.select.unselect') }}
      </UiButton>
```